server/enterprise/elasticsearch/elasticsearch/elasticsearch.go

```diff
@@ -756,7 +756,7 @@ func (es *ElasticsearchInterfaceImpl) IndexChannel(rctx request.CTX, channel *mo
 			Id_:    model.NewPointer(searchChannel.Id),
 		}, searchChannel)
 		if err != nil {
-			return model.NewAppError("Elasticsearch.IndexPost", model.NoTranslation, nil, "", http.StatusInternalServerError).Wrap(err)
+			return model.NewAppError("Elasticsearch.IndexChannel", model.NoTranslation, nil, "", http.StatusInternalServerError).Wrap(err)
```

Comment:
Good catches. I guess this was you, not aider?

Reply:
Yep, me